### PR TITLE
Fixed a bug in CreateCombatDamagedObjectSitRep 

### PR DESCRIPTION
**Pull request by TheSilentOne1**
Colony_Bombarded Sitrep only showed "colony_bombarded_own.png". Reason is that
planet->Unowned() always returns false.
planet->Owner()!=empire_id is the correct check.

### CODE REVIEW

**Comment by geoffthemedio** [comment]
What do you mean by planet->Unowned() always returns false? The function actually always returns false, or just when called in this bit of code for some reason related to what has to happen for that code to be called?

**Comment by TheSilentOne1** [comment]
Sorry for being inprecise; I mean only in this context. A planet that can be bombarded is never UnOwned(), because it is owned either by the enemy or yourself. Instead the code needs to check if the planet belongs to you or the enemy.

File: util/SitRepEntry.cpp [hunk 1]
```diff
@@ -180,7 +180,7 @@ SitRepEntry CreateCombatDamagedObjectSitRep(int object_id, int combat_system_id,
         sitrep.AddVariable(VarText::DESIGN_ID_TAG,     boost::lexical_cast<std::string>(ship->DesignID()));
 
     } else if (TemporaryPtr<const Planet> planet = boost::dynamic_pointer_cast<const Planet>(obj)) {
-        if (planet->Unowned())
+        if (planet->Owner()!=empire_id)
```
Comment (geoffthemedio) [inline util/SitRepEntry.cpp:183]: I think this is fine as it was. If the test passes, it creates an unonwed system bombarded message. Why should it do this for planets owned by another empire?

Perhaps you want to create a new type of sitrep for other empires' planets being bombarded?

Also, put spaces around the !=

**Comment by geoffthemedio** [comment]
Why does this pull request also include a bunch of new icon additions? Was that intended?

**Comment by TheSilentOne1** [comment]
I had intended to create a different pull request for the icons - didn't realize they'd be added. If you want I can try to revert the commit and make a second pull request.

> Perhaps you want to create a new type of sitrep for other empires' planets being bombarded?

This is already implemented, but not working. If a colony of the player is bombarded a different sitrep message / icon will be created then if an enemy's colony is affected; i. e. there's a message "my planet was bombarded" and a message "our enemy's planet was bombarded". However, only the first message was created; this was not very obvious because of the message's wording, and I only noticed it because only the "my planet was bombarded"-icon would show.
My patch fixes this.

**Comment by geoffthemedio** [comment]
The point of the check is to pick which sitrep template to use; one is for planet that are owned, one is for unowned planets, and that's what the check tests. I don't understand why you want to use the unowned planet sitrep for other empire planets. The owned planet sitrep template covers both player owned and other empire owned planets being bombarded.

**Comment by Dilvish-fo** [comment]
> I mean only in this context. A planet that can be bombarded is never UnOwned(), because it is owned either by the enemy or yourself. 

@TheSilentOne1 It sounds to me like you mistunderstand "owned" in the FreeOrion universe.  "Owned" means owned by an Empire; a planet not owned by an empire, but populated by natives (and thus, I believe, subject to being bombarded), would still be "UnOwned".

Of course, you might still be correct about planet->Owner()!=empire_id being the correct check in this case,  but I wanted to hopefully clear things up about ownership.

**Comment by TheSilentOne1** [comment]
> [Geoff] The owned planet sitrep template covers both player owned and other empire owned planets being bombarded.
> 
> [Dilvish] It sounds to me like you mistunderstand "owned" in the FreeOrion universe. "Owned" means owned by an Empire; a planet not owned by an empire, but populated by natives (and thus, I believe, subject to being bombarded), would still be "UnOwned".

Ah, this clears things up, so I will close this pull request now. Maybe I will get back to creating individual sitrep entries for bombardment of player and non-player owned planets at some point after 0.4.5. Sorry, thanks for your help.

**Comment by MatGB** [comment]
There is/was a sitrep in en.txt for own planet bombarded that was separate from somoene else's or unowned. I'd like for that to happen for a variety of reasons, one of which is I really want to know if my planets have been attacked, I'm not bothered at all by the list of sitreps basically telling me my fleets are doing their jobs. But it's a lower priority.

**Pull request closed without merging**